sdk/keyvault/keyvault-secrets/src/index.ts

```diff
@@ -346,23 +352,21 @@ export class SecretClient {
   public async updateSecretProperties(
     secretName: string,
     secretVersion: string,
-    options?: UpdateSecretOptions
-  ): Promise<Secret> {
+    options?: UpdateSecretPropertiesOptions
+  ): Promise<SecretProperties> {
     if (options) {
       const unflattenedProperties = {
         enabled: options.enabled,
         notBefore: options.notBefore,
-        expires: options.expires
+        expires: options.expiresOn
```

Comment:
`expiresOn`. Please make sure the other properties that are exported are renamed suitably.

Reply:
this is what gets sent to the service